python/lsst/ts/observatory/control/remote_group.py

```diff
@@ -372,52 +479,117 @@ async def check_comp_heartbeat(self, component):
         """
 
         while True:
-            await getattr(self.rem, component).evt_heartbeat.next(
+            await getattr(self.rem, self._components[component]).evt_heartbeat.next(
                 flush=True, timeout=self.fast_timeout
             )
 
+    async def inspect_settings(self, components=None):
+        """Return available settings.
+
+        Inspect `settingVersions` from CSCs and return dictionary with all the
+        available settings.
+
+        Parameters
+        ----------
+        components : `list` of `str` or `None`
+            List of CSC names from the group to inspect settings. If `None`
+            inspect all components in the group. The name of the CSC must be as
+            it appears in the `components` attribute, which is the name of
+            the CSC in lowercase, replacing ":" by "_" for indexed components,
```

Comment:
Please fix this. The name as it appears in the `components` attribute is Hexapod:1 not hexapod_1.

Please look for other instances as well -- perhaps search for ``` `components` attribute```?